stock_cycle_count/models/stock_warehouse.py

```diff
@@ -0,0 +1,145 @@
+# Copyright 2017-18 ForgeFlow S.L.
+#   (http://www.forgeflow.com)
+# License AGPL-3.0 or later (https://www.gnu.org/licenses/agpl.html).
+
+import logging
+from datetime import datetime, timedelta
+
+from odoo import api, fields, models
+
+_logger = logging.getLogger(__name__)
+
+
+class StockWarehouse(models.Model):
+    _inherit = "stock.warehouse"
+
+    cycle_count_rule_ids = fields.Many2many(
+        comodel_name="stock.cycle.count.rule",
+        relation="warehouse_cycle_count_rule_rel",
+        column1="warehouse_id",
+        column2="rule_id",
+        string="Cycle Count Rules",
+    )
+    cycle_count_planning_horizon = fields.Integer(
+        string="Cycle Count Planning Horizon (in days)",
+        help="Cycle Count planning horizon in days. Only the counts inside "
+        "the horizon will be created.",
+    )
+    counts_for_accuracy_qty = fields.Integer(
+        string="Inventories for location accuracy calculation",
+        default=1,
+        help="Number of latest inventories used to calculate location " "accuracy",
+    )
+
+    def get_horizon_date(self):
+        self.ensure_one()
+        date = datetime.today()
+        delta = timedelta(self.cycle_count_planning_horizon)
+        date_horizon = date + delta
+        return date_horizon
+
+    @api.model
+    def _get_cycle_count_locations_search_domain(self, parent):
+        domain = [
+            ("parent_path", "=like", parent.parent_path + "%"),
```

Comment:
Why not using 'child of' operator ?

Reply:
Question not answered :smiley:

Reply:
I hadn't considered that, but it seems that using 'child of' is cleaner, changed.